opentracing-cpp-devel.spec

```diff
@@ -0,0 +1,46 @@
+Name:           %{_name}
```

Comment:
Aren't these spec files generated by cpack?

Reply:
That was my initial commit using those, I'll delete those.